scripts/get-env-root.sh

```diff
@@ -30,6 +30,21 @@ apt-get install -y iverilog gtkwave
 # Nothing needed
 
 # libfpgalink
+
```

Comment:
If they are already set in my environment sudo -E will fix the problem. If we assume that proxy variables are already set then we dont need to prompt for proxy.